### PR TITLE
setup design system as a new package

### PR DESCRIPTION
**Pull request by krzysu**
I added a new package where I'm using Chakra as a base for our Meta design system. I set up storybook there and added fonts for now. Let me know what do you think. 
I will continue with colors and custom components but I would like to merge this already.

image

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -130,13 +130,36 @@
   dependencies:
     "@apollo/react-common" "^3.2.0-beta.1"
 
+"@babel/code-frame@7.5.5":
```
Comment (krzysu) [inline yarn.lock:133]: I don't know why there are some many changes in yarn.lock here, but that is the result of running yarn in root folder of this repo on my machine.

File: package.json [hunk 2]
```diff
@@ -20,8 +20,8 @@
     "hasura:migrate:apply": "npm run hasura migrate apply",
     "test": "lerna run test --parallel",
     "clean": "lerna clean",
-    "format": "prettier --ignore-path .gitignore --write \"{*,**/*}.{ts,tsx,js,json,yml,yaml,md}\"",
-    "lint": "eslint --ignore-path .gitignore \"./packages/**/*.{ts,tsx,js}\"",
+    "format": "prettier --ignore-path .gitignore --write \"{*,**/*}.{ts,tsx,js,jsx,json,yml,yaml,md}\"",
+    "lint": "eslint --ignore-path .gitignore \"./packages/**/*.{ts,tsx,js,jsx}\"",
```
Comment (krzysu) [inline package.json:24]: I don't want to write storybook stories in typescript, doesn't bring any value

File: packages/design-system/src/theme.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { theme } from '@chakra-ui/core';
+
+export const metaTheme = {
```
Comment (hammadj) [inline packages/design-system/src/theme.js:3]: Source files should be in TS, even if storybooks arent

File: packages/design-system/stories/0-Text.stories.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
+import { Stack, Text } from '@chakra-ui/core';
+import React from 'react';
+
+// eslint-disable-next-line import/no-default-export
```
Comment (hammadj) [inline packages/design-system/stories/0-Text.stories.jsx:4]: Should add a ESLint override for all stories so you dont need to do this in every file. 

Can make a .eslintrc file in the design-system/stories folder with this: 
```json
{
    "rules": {
      "import/no-default-export": "off",
      "@typescript-eslint/no-explicit-any": "off"
    }
}
```

**Pull request merged**